### ISSUE TITLE
Pagination and symple form dont work by default.

### ISSUE THREAD

**Issue by MProG** [datarockets/rails-views#10]
add   
  `  include ::Cell::Slim

  include ::Rails.application.routes.url_helpers

  include ActionView::Helpers::DateHelper
  include ActionView::Helpers::FormOptionsHelper
  include ActionView::Helpers::UrlHelper
  include ActionView::Helpers::CaptureHelper
  include ActionView::Helpers::OutputSafetyHelper
  include ActionView::Helpers::TranslationHelper
  include ActionDispatch::Http::Cache::Request
  include ActionView::RecordIdentifier
  include WillPaginate::ActionView
  include SimpleForm::ActionViewExtensions::FormHelper

  include ::Cell::Builder

  include AbstractController::Helpers

  def render_each_and_join(views)
    views.inject("") { |partials, view| partials << render("partials/#{view}") }
  end

  def pagination
    will_paginate(model, renderer: BootstrapPagination::Rails)
  end

  def url_for(options)
    parent_controller.url_for(options)
  end

  def self.option(*array)
    array.each do |option|
      define_method(option) do
        options[option]
      end
    end
  end`

**Comment by DDKatch** [comment]
@roman-dubrovsky @AleksSenkou maybe it's better to just update readme for `kaminari`, `will_paginate` and `pagy` gems instead of adding support for every one of them. Going to check issue with *simple_form*

**Comment by roman-dubrovsky** [comment]
@DDKatch yeah, I agree with you. Adding default supporting for some gems looks like bullshit

**Comment by AleksSenkou** [comment]
@DDKatch sounds reasonable: including by default isn't a good thing to go with 👍 

We can add info to README for now, 
as the second thing we can add initializer and include stuff there (we can start discussion about it in separate issue)
